src/main/java/jbash/Main.java

```diff
@@ -14,11 +19,14 @@
 public class Main {
     private static final Scanner userIn = new Scanner(System.in);
     private static final boolean debug = false;
+    private static FileSystemAPI FSAPI = FileSystemAPI.getInstance();
 
     public static void main(String[] args) {
+
         while (true) {
             // Prompt
-            System.out.print("[jbash]> ");
+            System.out.println("[jbash] " + (FSAPI.getCurrentDirectory().getPath()));
```

Comment:
Good start for the prompt, but this should probably be one line (e.g. `[jbash] <pwd> $`).
I would probably want to abstract this out into its own module so down the line the user could modify it (think changing $PS1 in your `.bashrc`).